### ISSUE TITLE
Synchronize database doesn't seem to be working

### ISSUE THREAD

**Issue by PhilippeDeSousa** [nestjs/typeorm#709]
## I'm submitting a...
<pre><code>
[ ] Regression 
[X] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior
I'm trying to "reset" my database when running my tests. I'm currently using `synchronize`, but it doesn't seem to work.
This is what my app.module.ts looks like:

<pre><code>
@Module({
	imports: [TypeOrmModule.forRoot({
		type: 'postgres',
		host: 'localhost',
		port: 35432,
		username: 'user',
		password: 'pass',
		database: 'db',
		entities: [join(__dirname, '../../config/database/entities/*.entity.{ts,js}')],
		synchronize: true
	}),
		UserModule
	],
	controllers: [AppController],
	providers: [AppService],
})
export class AppModule {};
</code></pre>

When I run my tests for the first time, the register works fine, because no user is found. When I run my tests a second time, it fails because the user still exists in the database (even though I'm using synchronize when connecting to the database).
## Expected behavior
I'm expecting my database to be cleared when I connect to it using `synchronize`

## Reproducing
My code is available here: https://github.com/24school/API.
Basically, I run docker-compose up postgres to launch my database, and then I run yarn test twice. It works the first time, but not the second because my database wasn't reset.

Thanks a lot for the help!

**Comment by kamilmysliwiec** [comment]
Please, report this issue in the typeorm/typeorm repository.
